src/pages/Mavanimes/main.ts

```diff
@@ -0,0 +1,59 @@
+import { pageInterface } from '../pageInterface';
+
+export const Mavanimes: pageInterface = {
+  name: 'Mavanimes',
+  domain: 'http://www.mavanimes.co/',
+  languages: ['French'],
+  type: 'anime',
+  isSyncPage(url) {
+    if (j.$('select').length) {
+      return true;
+    }
+    return false;
+  },
+  
+
+  sync: {
+    getTitle(url) {
+      var title = j.$('meta[property="article:section"]').attr('content');
+      if (!title) {
+        return '';
+      }
+
+
+      return title.slice(0, -7);
+    },
+    getIdentifier(url) {
+      return Mavanimes.sync.getTitle(url);
+    },
+    getOverviewUrl(url) {
```

Comment:
I dont know why but not all animes have an overview page.
http://www.mavanimes.co/kaizoku-oujo-03-vostfr/